src/network/networkd-json.c

```diff
@@ -1036,6 +1037,52 @@ static int dhcp_server_append_json(Link *link, JsonVariant **v) {
         return json_variant_set_field_non_null(v, "DHCPServer", w);
 }
 
+static int dhcp6_get_vendor_options_append_json(Link *link, sd_dhcp6_option *option, JsonVariant **ret) {
```

Comment:
This is mostly a trivial wrapper of `json_variant_append_arrayb()`. Please merge this with `dhcp6_client_vendor_options_append_json()`.

Reply:
Removed and used json_variant_append_arrayb()